backend/tests/test_ingest_statements.py

```diff
@@ -0,0 +1,90 @@
+from composer.services.cs_ingestion.cs_ingestion_services import ingest_statements
+from composer.models import ConnectivityStatement
+import unittest
+from composer.services.cs_ingestion.neurondm_script import log_error
+from composer.enums import CSState
+from django.db.models import Q
+
+
+class TestIngestStatements(unittest.TestCase):
+    """
+    NOTE:
+    This test depends on the directory system of the Scicrunch neurondm - here - https://raw.githubusercontent.com/SciCrunch/NIF-Ontology/neurons/**/*.ttl
+    Check more details in neurondm_script.py
+    """
+
+    def flush_connectivity_statements(self):
+        ConnectivityStatement.objects.all().delete()
+
+    def test_full_and_label_import(self):
```

Comment:
suggestion (non-blocking): This test seems to be doing a lot of ingestions. Would it make sense to split in different tests? One for each?

Reply:
Done, makes it better yes thanks!